isr_exploration/CTE_Thurs.ipynb

```diff
@@ -0,0 +1,332 @@
+    "            if name == 'amp':\n",
+    "                return_data[name][i] = int(arr[index][4])\n",
+    "            else:\n",
+    "                return_data[name][i] = float(arr[index][4])\n",
+    "    return return_data\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def findFitsFiles(basedir, rawname, lsst_num):\n",
+    "    # Finds the right fits files in the repo dir and returns the visit number\n",
+    "    low_counts = 1000\n",
+    "    high_counts = 100000\n",
+    "    base = basedir + 'raw/'+rawname+'/'\n",
+    "    for f in os.listdir(base):\n",
+    "        fits_file = fits.open(os.path.join(base, f))\n",
+    "        fits_lsst_num = fits_file[0].header['LSST_NUM']\n",
+    "        if fits_lsst_num != lsst_num: continue\n",
+    "        val = fits_file[1].header['AVERAGE']\n",
+    "        bias = fits_file[1].header['AVGBIAS']\n",
+    "        val = val - bias\n",
+    "        if val > low_counts:\n",
+    "            low_file = int(f.split('-')[0])\n",
+    "            low_counts = 1E7\n",
+    "        if val > high_counts:\n",
+    "            high_file = int(f.split('-')[0])\n",
+    "            high_counts = 1E7\n",
+    "    return (low_file, high_file)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# E2V\n",
+    "# These values are for the E2V detector\n",
+    "\n",
+    "E2V_BOOTCAMP_REPO_DIR= '/project/bootcamp/repo_RTM-007/'\n",
+    "rawname = '7086'\n",
+    "lsst_num = 'E2V-CCD250-175'\n",
+    "E2V_OFFSET = 10\n",
+    "E2V_Xsize = 512\n",
+    "flipXY = {'C00':(False, True),\n",
+    "          'C01':(False, True),\n",
+    "          'C02':(False, True),\n",
+    "          'C03':(False, True),\n",
+    "          'C04':(False, True),\n",
+    "          'C05':(False, True),\n",
+    "          'C06':(False, True),\n",
+    "          'C07':(False, True),\n",
+    "          'C10':(True, False),\n",
+    "          'C11':(True, False),\n",
+    "          'C12':(True, False),\n",
+    "          'C13':(True, False),\n",
+    "          'C14':(True, False),\n",
+    "          'C15':(True, False),\n",
+    "          'C16':(True, False),\n",
+    "          'C17':(True, False)\n",
+    "         }\n",
+    "(E2V_low_file, E2V_high_file) = findFitsFiles(E2V_BOOTCAMP_REPO_DIR, rawname, lsst_num)\n",
+    "filename = '/home/cslage/bootcamp-work/isr_exploration/E2V-CCD250-175_CTE.txt'\n",
+    "E2V_cte_data = readEtravelerCTE(filename)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#ITL\n",
+    "# These values are for the ITL detector\n",
+    "\n",
+    "ITL_BOOTCAMP_REPO_DIR= '/project/bootcamp/repo_RTM-011/'\n",
+    "rawname = '5943D'\n",
+    "lsst_num = 'ITL-3800C-083-Dev'\n",
+    "ITL_OFFSET = 20\n",
```

Comment:
I kind of expected a lot of these sensor sizes and related flipXY concepts to be defined somewhere inside of the stack (perhaps the mapper), but for now I think this is fine.